tests/common.c

```diff
@@ -442,18 +436,19 @@ static void read_arguments(int *_argc, char*** _argv, int* iparam)
 
 static void parse_arguments(int *iparam) {
     int verbose = iparam[IPARAM_RANK] ? 0 : iparam[IPARAM_VERBOSE];
+    char *value;
+    int rc;
 
     /* we want to run at least once, right? */
     if(iparam[IPARAM_NRUNS] < 1) iparam[IPARAM_NRUNS] = 1;
 
-    if(iparam[IPARAM_NGPUS] < 0) iparam[IPARAM_NGPUS] = 0;
-    if(iparam[IPARAM_NGPUS] > 0) {
-        if (iparam[IPARAM_VERBOSE] > 3) {
-            parsec_setenv_mca_param( "device_show_capabilities", "1", &environ );
-        }
-        if (iparam[IPARAM_VERBOSE] > 2) {
-            parsec_setenv_mca_param( "device_show_statistics", "1", &environ );
-        }
+    if(iparam[IPARAM_NGPUS] == DPLASMA_ERR_NOT_SUPPORTED) iparam[IPARAM_NGPUS] = 0;
+    rc = asprintf(&value, "%d", iparam[IPARAM_NGPUS]); (void)rc;
+    parsec_setenv_mca_param( "device_cuda_enabled", value, &environ );
+    free(value);
+    if(iparam[IPARAM_NGPUS] && iparam[IPARAM_VERBOSE] >= 4) {
```

Comment:
This is a behavior change as the statistics and capabilities are linked together by the verbosity level. I like being able to see just the statistics without seeing the capabilities.

Reply:
Undid this